src/engines/kotor/script/functions_object.cpp

```diff
@@ -72,6 +72,48 @@ void Functions::getObjectByTag(Aurora::NWScript::FunctionContext &ctx) {
 	ctx.getReturn() = search->get();
 }
 
+void Functions::getMinOneHP(Aurora::NWScript::FunctionContext &ctx) {
+	Aurora::NWScript::Object *object = ctx.getParams()[0].getObject();
+
+	Object *kotorObject = ObjectContainer::toObject(object);
+
+	ctx.getReturn() = kotorObject->getMinOneHitPoints();
```

Comment:
You need to check for kotorObject being 0 before dereferencing.

`ctx.getReturn() = kotorObject ? kotorObject->getMinOneHitPoints() : 0`

should do it.